contributors/devel/bazel.md

```diff
@@ -55,17 +70,21 @@ To update `BUILD` files, run:
 $ ./hack/update-bazel.sh
 ```
 
-**NOTE**: `update-bazel.sh` only works if check out directory of Kubernetes is `$GOPATH/src/k8s.io/kubernetes`.
+To prevent Go rules from being updated, consult the [gazelle
+documentation](https://github.com/bazelbuild/rules_go/tree/master/go/tools/gazelle).
 
-Only rules which are automanaged will be updated, but all rules will be
-auto-formatted.
+Note that BUILD files style rules, and running `update-bazel.sh` will
```

Comment:
This doesn't make sense, consider rewording.

Reply:
I apparently can't english today. thanks!